NachoSpigot-Server/src/main/java/org/bukkit/craftbukkit/Main.java

```diff
@@ -20,18 +20,6 @@
     public static boolean useConsole = true;
 
     public static void main(String[] args) {
-//        [Nacho-0034]
-//        Remove this message from TacoSpigot, this won't work on Java 8 or higher because JAVA_1_8 doesn't exist on the JavaVersion enum when using these versions. (I have no clue why)
-//        So what was the point of making it support Java 14 if it never worked because of this ¯\_(ツ)_/¯
-//        Changed Java 8 message to Java 15.
-//        ---------------------------------------------------------------
-//        if (!SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_1_8)) {
-//            System.err.println("TacoSpigot requires java 8");
-//            System.err.println("Oracle dropped all support for java " + SystemUtils.JAVA_VERSION);
-//            System.err.println("Please update to use TacoSpigot and the numerous bug-fixes, performance improvements, and security fixes");
-//            System.err.println("Shutting down");
-//            System.exit(1);
-//        }
         try {
             if(!SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_15)) {
```

Comment:
Should it be changed to java 16 because protocollib works fine?

Reply:
We *could*, yes. But Java 15 is recommended either way.